less/home-page.less

```
@@ -58,18 +81,33 @@
       }
 
       .btn {
+        .button(10rem, 3.5rem, 2rem);
+        @media(max-width: @mobile) {
+          .margins-mobile();
```

This stretches it out to almost full width on mobile. Should be responsive as well. 
```suggestion
   padding: 15px 43%;
```